java/downgrades/81ccfabe82e696953268e784979262e56871ce86/upgrade.properties

```diff
@@ -0,0 +1,3 @@
+description: Remove ktDataClasses relation
```

Comment:
The SHA of this folder is probably not good. And maybe the old and new schemas need to be swapped.